### ISSUE TITLE
Same price bid allowed

### ISSUE THREAD

**Issue by AlexanderBeck0** [AlexanderBeck0/auction-house#63]
When buyer is placing a bid, it lets them place the same price bid as the previous one.

![image](https://github.com/user-attachments/assets/392dc40a-81c5-40c1-aee4-f5bf7d57b378)

**Comment by AlexanderBeck0** [comment]
Also, after trying to outbid myself, I got an error but it still added the bid

**Comment by AlexanderBeck0** [comment]
See if using this helps:
```
// account.balance >= item.price + totalBidCost
if (event.bid < item.price || (item.initialPrice !== item.price && event.bid <= item.price)) {
  return { statusCode: 400, error: "Must increase the bid on the item!" }
}
```
It currently uses the `event.bid < item.initialPrice`, but it should be `event.bid < item.price`.

**Comment by AlexanderBeck0** [comment]
Fixed in b9c149bf9dc9ee015ef83b892148227479fbd423
